hal/src/msom/pinmap_defines.h

```diff
@@ -110,6 +110,9 @@
 #define UNUSED_PIN1         44
 #define UNUSED_PIN2         45
 
+// EVT
+#define LOW_BAT_DEPRECATED  A5
```

Comment:
Just a reminder that we should remove this in the future after PVT?

Reply:
Yes, definitely want to get rid of this ASAP